# Patch from Debian by Thomas Schmidt <thomas.schmidt@in.stud.tu-ilmenau.de>
# Adapted for GeeXboX by Davide Cavalca <davide@geexbox.org>
diff -urNad --exclude=CVS --exclude=.svn ./menu.c /tmp/dpep-work.UjgjLU/vdr-plugin-osdteletext-0.5.1/menu.c
--- ./menu.c	2005-08-12 13:24:18.000000000 +0200
+++ /tmp/dpep-work.UjgjLU/vdr-plugin-osdteletext-0.5.1/menu.c	2005-08-16 18:50:04.000000000 +0200
@@ -21,7 +21,7 @@
 #include "txtrecv.h"
 
 
-#define TXTROOT "/vtx"
+#define TXTROOT "/tmp/vtx"
 
 #define GET_HUNDREDS(x) ( ( (x) - ((x)%256) ) /256 )
 #define GET_TENS(x)  ( (( (x) - ((x)%16) )%256 ) /16 )
diff -urNad --exclude=CVS --exclude=.svn ./osdteletext.c /tmp/dpep-work.UjgjLU/vdr-plugin-osdteletext-0.5.1/osdteletext.c
--- ./osdteletext.c	2005-08-16 16:19:25.000000000 +0200
+++ /tmp/dpep-work.UjgjLU/vdr-plugin-osdteletext-0.5.1/osdteletext.c	2005-08-16 18:50:04.000000000 +0200
@@ -121,7 +121,8 @@
   // Return a string that describes all known command line options.
   return "  -d        --directory=DIR    The directory where the temporary\n"
          "                               files will be stored.\n"
-         "                               (default: /vtx, recommended: /tmp/vtx\n"
+         "                               (default: /tmp/vtx),\n"
+         "                                recommended: /tmp/vtx\n"
          "                                or /var/cache/vdr/osdteletext.)\n"
          "                               Ensure that the directory exists and is writable.\n"
          "  -n        --max-cache=NUM    Maximum size in megabytes of cache used\n"
diff -urNad --exclude=CVS --exclude=.svn ./README /tmp/dpep-work.UjgjLU/vdr-plugin-osdteletext-0.5.1/README
--- ./README	2005-08-16 16:24:33.000000000 +0200
+++ /tmp/dpep-work.UjgjLU/vdr-plugin-osdteletext-0.5.1/README	2005-08-16 18:50:20.000000000 +0200
@@ -76,7 +76,8 @@
    
      -d        --directory=DIR    The directory where the temporary
                                   files will be stored.
-                                  (standard value: /vtx, recommended: /tmp/vtx
+                                  (standard value: /tmp/vtx,
+                                   recommended: /tmp/vtx
                                    or /var/cache/vdr/osdteletext.)
                                   Ensure that the directory exists and is writable.
      -n        --max-cache=NUM    Maximum size in megabytes of cache used
diff -urNad --exclude=CVS --exclude=.svn ./README.DE /tmp/dpep-work.UjgjLU/vdr-plugin-osdteletext-0.5.1/README.DE
--- ./README.DE	2005-08-16 16:31:59.000000000 +0200
+++ /tmp/dpep-work.UjgjLU/vdr-plugin-osdteletext-0.5.1/README.DE	2005-08-16 18:50:34.000000000 +0200
@@ -63,7 +63,8 @@
    Verfügbare Optionen:
    
      -d        --directory=DIR    Das Verzeichnis für die temporären Dateien.
-                                  (Voreinstellung: /vtx, empfohlen: /tmp/vtx
+                                  (Voreinstellung: /tmp/vtx,
+                                   empfohlen: /tmp/vtx
                                    oder /var/cache/vdr/osdteletext.)
                                   Stellen Sie sicher, dass das Verzeichnis existiert
                                   und beschreibbar ist.
diff -urNad --exclude=CVS --exclude=.svn ./txtrecv.c /tmp/dpep-work.UjgjLU/vdr-plugin-osdteletext-0.5.1/txtrecv.c
--- ./txtrecv.c	2005-08-12 13:20:09.000000000 +0200
+++ /tmp/dpep-work.UjgjLU/vdr-plugin-osdteletext-0.5.1/txtrecv.c	2005-08-16 18:50:04.000000000 +0200
@@ -26,7 +26,7 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 
-const char *RootDir::root = "/vtx";
+const char *RootDir::root = "/tmp/vtx";
 
 void RootDir::setRootDir(const char *newRoot) {
    root=newRoot;
